<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QChartView" name="pieChart">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QChartView" name="BarUni">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>800</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QChartView" name="BarEmi">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>800</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelUni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Universal</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEmi">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EMI</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="uniView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>330</width>
      <height>350</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>350</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>350</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="emiView">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>70</y>
      <width>330</width>
      <height>350</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>350</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>350</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
